.github/workflows/build.yml

```diff
@@ -2,15 +2,43 @@ name: JavaScript
 
 on:
   push:
-    branches:
-      - master
+  pull_request:
 
 jobs:
   build:
+    name: Build JS
     runs-on: ubuntu-latest
 
     steps:
-      - uses: actions/checkout@master
-      - uses: flarum/action-build@master
+      - name: Check out code
+        uses: actions/checkout@master
```

Comment:
Should this be tagged

Reply:
Not now I removed it ;)

But the flarum action should 100% be tagged.

I want to add some `with` options to it to allow custom build scripts and toggling between npm and yarn, and those changes should be tagged.